### ISSUE TITLE
heimdalld doesn't sync no peers

### ISSUE THREAD

**Issue by blissend** [maticnetwork/heimdall#829]
**Heimdall version**: 0.2.9

**OS** (e.g. from /etc/os-release): Ubuntu 20.04.4 LTS

**What happened**:

Heimdall ran into issue described here https://github.com/maticnetwork/heimdall/issues/696 which I ran comands to fix and used seeds from https://forum.polygon.technology/t/heimdall-v0-2-7/2423

Now the logs show this after turning on debug mode...
```Jun 02 17:26:46 ip-172-31-25-111 heimdalld[172288]: D[2022-06-02|17:26:46.051] Blockpool has no peers                       module=blockchain
Jun 02 17:26:47 ip-172-31-25-111 heimdalld[172288]: D[2022-06-02|17:26:47.051] Consensus ticker                             module=blockchain numPending=0 total=0 outbound=0 inbound=0
Jun 02 17:26:47 ip-172-31-25-111 heimdalld[172288]: D[2022-06-02|17:26:47.051] Blockpool has no peers                       module=blockchain
Jun 02 17:26:48 ip-172-31-25-111 heimdalld[172288]: D[2022-06-02|17:26:48.050] Consensus ticker                             module=blockchain numPending=0 total=0 outbound=0 inbound=0
Jun 02 17:26:48 ip-172-31-25-111 heimdalld[172288]: D[2022-06-02|17:26:48.051] Blockpool has no peers                       module=blockchain
Jun 02 17:26:49 ip-172-31-25-111 heimdalld[172288]: D[2022-06-02|17:26:49.050] Consensus ticker                             module=blockchain numPending=0 total=0 outbound=0 inbound=0
```

**What you expected to happen**:

Sync to happen and catching up to be false

**Have you tried the latest version**: yes

--

**NOTE: Please don't give discord link and close ticket. The discord link doesn't work as it shows invalid or expired. Plus doesn't prevent as far as I know repeated questions for same fix.**

**Comment by ssandeep** [comment]
Hi @blissend,

If you have successfully resolved the AppHash error and are now facing peering issues, please try restarting heimdall and also check if you have configured the seed nodes for heimdall as per https://docs.polygon.technology/docs/develop/network-details/full-node-deployment/

We discuss issues/bugs in the codebase in github and for other issues like nodes not syncing/peering issues, which are generally to do with config/setup, we encourage people to post on discord as we have a dedicated team to help in such scenarios. The permanent link for joining discord is https://discord.gg/KeeAxrRbX6 and the channel is #full-node-queries

**Comment by blissend** [comment]
I've tried that but unfortunately no go. In the end I just terminated the server and recreated it which worked. Now I'm back in sync so I'll close ticket.

> We discuss issues/bugs in the codebase in github and for other issues like nodes not syncing/peering issues, which are generally to do with config/setup, we encourage people to post on discord as we have a dedicated team to help in such scenarios. The permanent link for joining discord is https://discord.gg/KeeAxrRbX6 and the channel is #full-node-queries

I would highly advise NOT using discord for troubleshooting/fixes except for high priority stuff. People browsing online are less likely to find out how others fixed something, contribute towards it and not as inclined to install yet another app. Plus it makes it more likely questions/requests for help get repeated unnecessarily since chat applications aren't ideal for this. I really think engagement would go a long way if using other solutions.
